### DISCUSSION TITLE
How to upload a file using update_items_content and a vector of bytes?

### DISCUSSION THREAD

**Discussion by samvv** [sreeise/graph-rs-sdk#464] [General]
I'm using `v2.0.0-beta.0`. I would like to make use of Drive's `update_items_content`, but by the looks of it, this method only supports setting a path to a local file and having that uploaded. I would like to upload small amount of arbitrary bytes. Is there a way?

I know of `create_upload_session` and I use it for large files, but I've noticed requests using `update_items_content` are usually a bit faster and I really need the performance.

**Comment by samvv** [comment]
Never mind I just found the answer in https://github.com/sreeise/graph-rs-sdk/blob/v2.0.0-beta.0/examples/drive/upload_file.rs. `BodyRead` is the answer, I think.

**Reply by sreeise** [reply]
No problem! Yep you found the way to use bytes.
